src/containers/table/Table.js

```diff
@@ -187,6 +187,30 @@ class TableView extends React.Component {
     });
   }
 
+  clearTags() {
+    const selected = this.state.selectedTags;
+    let unSelected = this.state.unSelectedTags;
+    for (let i = 0; i < selected.length; i++) {
+      unSelected.push(selected[i]);
```

Comment:
After using 'clear tags' tags won't be sorted alphabetically, sort them after you push elements back to unselected tags

Reply:
Should be fixed